promgen/templates/promgen/vue/silence_modal.html

```diff
@@ -0,0 +1,108 @@
+<div id="silenceModal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="silenceModalLabel">
+  <div class="modal-dialog modal-lg" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" @click="state.show = false" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="silenceModalLabel">New Silence</h4>
+      </div>
+      <div class="modal-body" style="padding:10px 30px;">
+        
+        <div>
+          <div v-for="message in globalMessages" :key="message.id">
+            <div :class="message.class" >
+              [[ message.message ]]
+              <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+            </div>
+            
+          </div>
+        </div>
+        
+        <form @submit.prevent="submit()">
+
+          <div class="row mb-4">
+            <div class="col-md-4">
+              <h5>
+                Start from
+              </h5>
+              <div>
+                <input v-model="form.startsAt" placeholder="2006-10-25 14:30" type='datetime-local' class="form-control" /> 
+              </div>
+            </div>
+            <div class="col-md-4">
+              <h5>
+                End
+              </h5>
+              <div>
+                <input v-model="form.endsAt" placeholder="2006-10-25 14:30" type='datetime-local' class="form-control" /> 
+              </div>
+            </div>
+            <div class="col-md-4">
+              <h5>
+                Duration
+              </h5>
+              <div>
+                <input v-model="form.duration" placeholder="1m/1h/etc" class="form-control" />
+              </div>
+            </div>           
+          </div>
+
+          <div class="row mb-4">
+            <div class="col-md-12">
+              <h5>
+                Target
+              </h5>
+              <div class="labels">
+                <ul class="list-inline promgen-labels-list">
+                  <li v-for="(value, label) in state.labels">
+                    <div class="promgen-label-target">
+                      <span>[[label]]=~[[value]]</span>
```

Comment:
I think this is somewhat incorrect. We are not always using `=~` matches for everything. I think only the `instance` field is a `=~` while others should generally be a regular `=` match 🤔

Reply:
A little buried but

https://github.com/line/promgen/blob/32f1bbd51b620411c4b4f1852f65f40bb90c087c/promgen/prometheus.py#L289-L296

The only time it is a `=~` is when our matcher ends with `*` and in Promgen that is generally only for our quick silence for `instance/host`